Dockerfile

```diff
@@ -9,6 +9,8 @@ RUN ./steamcmd.sh +login anonymous \
 +app_update 896660 \
 validate +exit
 
+USER 1000:1000
```

Comment:
1.  What is the issue with using the `steam` user that comes from the `steamcmd2` base image?  
1.  Mind adding a comment here explaining what this `USER 1000:1000` line does?

Reply:
It doesn't have the same IDs, so there is an ownership conflict with folders unless a steam user is also created on the server (or whatever machine). The USER variable represents the UUID and GUIDs, and I can add that comment.